docs/_assets/codepen.js

```diff
@@ -36,29 +36,12 @@ CodeBlockCodePen.prototype.MDLIBS = [
 CodeBlockCodePen.prototype.init = function() {
   'use strict';
 
-  for (var i = 0, len = this.htmlCodeBlocks.length; i < len; i++) {
-    var pre = this.htmlCodeBlocks[i];
+  for (var i = 0, len = this.codepenButtons.length; i < len; i++) {
```

Comment:
just say whatever to old browsers and use

for(let node of this.codepenButtons) { ... } and make @sebmck happy